rules-tests/EarlyReturn/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/if_in_closure.php.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Rector\Tests\EarlyReturn\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+class IfInClosure
+{
+    public function run()
+    {
+        Text::make(___('Type'), function () {
+            if ($this->_fieldType_ === 'wire' && $this->users->getRole() === 'correct-role') {
+                return ___('Found');
+            }
+            return Arr::get(Purses::getOptions(), $this->_fieldType_) ?? $this->_fieldType_;
+        });
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\EarlyReturn\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+class IfInClosure
+{
+    public function run()
+    {
+        Text::make(___('Type'), function () {
+            if ($this->_fieldType_ !== 'wire') {
+                return Arr::get(Purses::getOptions(), $this->_fieldType_) ?? $this->_fieldType_;
+            }
+            if ($this->users->getRole() !== 'correct-role') {
+                return Arr::get(Purses::getOptions(), $this->_fieldType_) ?? $this->_fieldType_;
+            }
+            return ___('Found');
+        });
```

Comment:
This is very hard to read. Could you simplify it to minimal code using only variables?

Just in case we'll have to debug it in the future, so we won't have to read so much complexity.

Reply:
implemented https://github.com/rectorphp/rector-src/pull/289/commits/7581195d3f492ac14392fc9d5c6b8b85f9ee82a4

Reply:
more simplified with only variables https://github.com/rectorphp/rector-src/pull/289/commits/97c2bc9324d33cd80d65d85ce45520255297c64d